eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/request/TpaCommand.java

```diff
@@ -46,14 +48,24 @@ void execute(Player player, @Arg Player target) {
             .placeholder("{PLAYER}", target.getName())
             .send();
 
-        this.noticeService
-            .create()
-            .player(target.getUniqueId())
-            .notice(translation -> translation.tpa().tpaReceivedMessage())
-            .placeholder("{PLAYER}", player.getName())
-            .send();
+        this.isIgnoring(target, player).then((isIgnoring) -> {
+            if (isIgnoring) {
+                return;
+            }
```

Comment:
No message

Reply:
In my opinion message in this case is useless because player finally getting teleport request